documentation/content/installanduninstall.rst

```diff
@@ -77,9 +77,15 @@ Package install
 
 Update
 ------
-Every time you want to get a new version of `kw`, just pull from this
-repository and install kw again (`./setup.sh -i` removes legacy files and installs
-new ones).
+
+- Using `kw self-update`:
+   The `kw self-update` feature allows you to update kw without having
+   to clone and manually install `kw`. For more information see :ref:`kw-self-update<self-update-doc>`.
```

Comment:
Maybe `to clone the project's repository (...)` instead of just `to clone (...)`. Don't forget to break this line if it becomes to long (i.e., **really** overflowing the 80 chars).